File: TableView/iOS-d6ea7e2c8b480117c8fcab1a2d76294942a3e81b/SharedCore/SharedCore/WebViewController.m
//  Copyright 2010 Jonathan Steele
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

#import "WebViewController.h"

@interface WebViewController ()

@end

@implementation WebViewController

- (id)init
{
	self = [super init];
	if (self) {
        // Custom initialization
		self.hidesBottomBarWhenPushed = YES;
    }
	return self;
}

#pragma mark -
#pragma mark View lifecycle

- (void)viewDidLoad
{
	[super viewDidLoad];

	self.view.backgroundColor = [UIColor whiteColor];
}

- (void)setUrl:(NSString *)urlString
{
	UIWebView *webView = [[UIWebView alloc] initWithFrame:CGRectZero];
	self.view = webView;
	webView.delegate = self;
	webView.scalesPageToFit = YES;

	// load URL
	NSURL *webUrl = [NSURL URLWithString:urlString];
	NSURLRequest *request = [NSURLRequest requestWithURL:webUrl];
	[webView loadRequest:request];
}

#pragma mark -
#pragma mark Memory management

- (void)didReceiveMemoryWarning {
    // Releases the view if it doesn't have a superview.
    [super didReceiveMemoryWarning];
    // Relinquish ownership any cached data, images, etc that aren't in use.
}

@end

File: TableView/iOS-d6ea7e2c8b480117c8fcab1a2d76294942a3e81b/Noles Football - Swift/Noles Football/WebViewController.swift
//
//  WebViewController.swift
//  Noles Football
//
//  Created by Jonathan Steele on 6/3/14.
//  Copyright (c) 2014 Jonathan Steele. All rights reserved.
//

import UIKit

class WebViewController: UIViewController
{
    init(nibName nibNameOrNil: String!, bundle nibBundleOrNil: NSBundle!)
    {
        super.init(nibName: nibName, bundle: nibBundleOrNil)
        // Custom initialization
        self.hidesBottomBarWhenPushed = true
    }

    override func viewDidLoad()
    {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
        view.backgroundColor = UIColor.whiteColor()
    }
    
    func setURL(url: String)
    {
        let webview: UIWebView = UIWebView(frame: CGRectZero)
        view = webview
        webview.scalesPageToFit = true
        
        let weburl : NSURL = NSURL.URLWithString(url)
        let request: NSURLRequest = NSURLRequest(URL: weburl)
        webview.loadRequest(request)
    }

    override func didReceiveMemoryWarning()
    {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    /*
    // #pragma mark - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepareForSegue(segue: UIStoryboardSegue?, sender: AnyObject?) {
        // Get the new view controller using [segue destinationViewController].
        // Pass the selected object to the new view controller.
    }
    */
}
